db/migrate/20221004080825_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,15 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+  def change
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_index :users, :confirmation_token, unique: true, algorithm: :concurrently
+    User.update_all confirmed_at: DateTime.now
```

Comment:
It's an antipattern to run changes in a migration. In this case you should create a rake task and run itdefault value

Reply:
I followed instruction from devise https://github.com/heartcombo/devise/wiki/How-To:-Add-:confirmable-to-Users